docs/source/target-query.rst

```diff
@@ -0,0 +1,121 @@
+target-query
+------------
+
+The **target-query** tool is one of the most prominent tools of Dissect. It allows you to
+**query a forensic image**
+to extract useful information from it. The basic usage format is:
+
+.. code-block:: console
+
+    $ target-query <TARGET> -f <FUNCTION_NAME>
+
+Here <TARGET> is the file that contains the forensic image and <FUNCTION_NAME> is the name
+of the function you wish to apply.
+
+Loading
```

Comment:
I would maybe not describe this functionality first after the general usage guide of target-query. I think the reader would be way more interested in figuring out which functions/plugins there are.

Reply:
That's already discussed on the front page. I think some of your comments may stem from the fact that github does not present the documentation in the correct order. Have you also seen the HTML output? I know it is annoying to switch between the HTML and github (to comment), but the view on github might be misleading sometimes.